### PR TITLE
fix: Allow MQTT broker container config

### LINKED ISSUE #502: Allow overriding of mosquitto broker container image
### Description

Customer is running "offline" so all containers need to be loaded from a private docker registery

Currently the mosquitto go-auth container is hard coded, needs an override e.g. `forge.broker.image`

Also need to check if emqx can be likewise overriden

### Requested By

- https://app-eu1.hubspot.com/contacts/26586079/company/10459248347

### PR DESCRIPTION
**Pull request by hardillb**
fixes #502

## Description

Allow override on container image (to use local cache)

## Related Issue(s)

#502 502

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: helm/flowforge/templates/broker.yaml [hunk 1]
```diff
@@ -146,11 +144,5 @@ spec:
     protocol: TCP
     name: mqtt-ws
   selector:
-    {{- if and ( eq .Values.forge.broker.enabled true) ( eq .Values.forge.broker.teamBroker.enabled false ) -}}
     {{- include "forge.brokerSelectorLabels" . | nindent 4 }}
-    {{- else -}}
-    apps.emqx.io/db-role: core
-    apps.emqx.io/instance: emqx
-    apps.emqx.io/managed-by: emqx-operator
-    {{- end }}
```
Comment (ppawlowski) [inline helm/flowforge/templates/broker.yaml:155]: This one overwrites the quick fix we made yestarday.

Comment (hardillb) [reply helm/flowforge/templates/broker.yaml:155]: doh, must have forgot to pull first, will rebase

File: helm/flowforge/templates/broker.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ spec:
         {{- toYaml .Values.forge.broker.podSecurityContext | nindent 8 }}
       containers:
       - name: broker
-        image: iegomez/mosquitto-go-auth
+        image: {{ .Values.forge.broker.image | default "iegomez/mosquitto-go-auth" | quote }}
```
Comment (ppawlowski) [inline helm/flowforge/templates/broker.yaml:39]: The values file defines a default, static value. There is no need to make it redundant with the `default` function.
Also, the image value doesn't have to be quoted.

```suggestion
        image: {{ .Values.forge.broker.image }}
```

**Review by ppawlowski — APPROVED** [review]

**Comment by ppawlowski** [comment]
The failure in the lint workflow is acceptable due to the known error with the chart linting tool not using the new values file.

**Comment by hardillb** [comment]
Fixes for broker service look good to me.

**Pull request merged**